Source/Lib/Fluxor/ActionSubscriber.cs

```diff
@@ -91,6 +96,12 @@ public void UnsubscribeFromAllActions(object subscriber)
 						.Except(instanceSubscriptions)
 						.ToList();
 				}
+				
+				foreach (object subscription in subscribedInstances)
+				{
+				    if (SubscriptionsForInstance.ContainsKey(subscription))
+					SubscriptionsForInstance.Remove(subscription);
```

Comment:
You don't need to use ContainsKey before Remove as Remove will fail silently when the key does not exist and return `false` which we can ignore.  Also, please ensure the indentation is tabs only, not spaces.